<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="HiUpTestSuite" time="34" tests="2" failures="0" errors="0">
   <testsuite name="HiUpTestSuite" tests="2" failures="0" errors="0" time="34" timestamp="2018-07-02 10:07:51" hostname="Acer - DESKTOP-O5SIUU5" id="Test Suites/HiUpTestSuite">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="C:\\Users\\Acer\\Katalon Studio\\HiUpAutomation\\Reports\\HiUpTestSuite\\20180702_100740"/>
         <property name="logFiles" value="C:\\Users\\Acer\\Katalon Studio\\HiUpAutomation\\Reports\\HiUpTestSuite\\20180702_100740\\appium.log, C:\\Users\\Acer\\Katalon Studio\\HiUpAutomation\\Reports\\HiUpTestSuite\\20180702_100740\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="Acer - DESKTOP-O5SIUU5"/>
         <property name="hostAddress" value="192.168.210.2"/>
         <property name="os" value="Windows 10 64bit"/>
         <property name="katalonVersion" value="5.4.2.1"/>
      </properties>
      <testcase name="Test Cases/HiUP_LoginTest" classname="Test Cases/HiUP_LoginTest" status="PASSED">
         <system-out><![CDATA[2018-07-02 10:07:52 - [TEST_CASE][PASSED] - Test Cases/HiUP_LoginTest: Test Cases/HiUP_LoginTest

2018-07-02 10:07:52 - [TEST_STEP][FAILED] - Statement - Try: Unable to start app at: 'C:\Users\Acer\Desktop\HiUp_uat_apk\HIUP-2.2.2(68).apk' (Root cause: com.kms.katalon.core.appium.exception.AppiumStartException: Fail to start Appium server in 30 seconds)

2018-07-02 10:07:52 - [TEST_STEP][PASSED] - Statement - email = "jmeter12345@gmail.com": null

2018-07-02 10:07:52 - [TEST_STEP][PASSED] - Statement - pass = "y@nzEE2018": null

2018-07-02 10:07:52 - [TEST_STEP][FAILED] - startApplication: Unable to start app at: 'C:\Users\Acer\Desktop\HiUp_uat_apk\HIUP-2.2.2(68).apk' (Root cause: com.kms.katalon.core.appium.exception.AppiumStartException: Fail to start Appium server in 30 seconds)

2018-07-02 10:08:24 - [MESSAGE][FAILED] - Unable to start app at: 'C:\Users\Acer\Desktop\HiUp_uat_apk\HIUP-2.2.2(68).apk' (Root cause: com.kms.katalon.core.appium.exception.AppiumStartException: Fail to start Appium server in 30 seconds)

2018-07-02 10:08:24 - [TEST_STEP][PASSED] - Statement - Catch (Exception e): null

2018-07-02 10:08:24 - [TEST_STEP][PASSED] - Statement - out.println("Unable to login"): null

2018-07-02 10:08:24 - [TEST_STEP][PASSED] - Statement - out.println(e.getMessage()): null]]></system-out>
         <system-err><![CDATA[]]></system-err>
      </testcase>
      <testcase name="Test Cases/JobFeedTest" classname="Test Cases/JobFeedTest" status="PASSED">
         <system-out><![CDATA[2018-07-02 10:08:24 - [TEST_CASE][PASSED] - Test Cases/JobFeedTest: Test Cases/JobFeedTest

2018-07-02 10:08:24 - [TEST_STEP][PASSED] - Statement - Try: Timeout '0' is invalid. Using default page load timeout: '30'

2018-07-02 10:08:24 - [TEST_STEP][PASSED] - tap: Timeout '0' is invalid. Using default page load timeout: '30'

2018-07-02 10:08:24 - [MESSAGE][WARNING] - Timeout '0' is invalid. Using default page load timeout: '30'

2018-07-02 10:08:25 - [TEST_STEP][PASSED] - Statement - Catch (Exception e): null

2018-07-02 10:08:25 - [TEST_STEP][PASSED] - Statement - out.println("Unable to test Job Feed Menu."): null

2018-07-02 10:08:25 - [TEST_STEP][PASSED] - Statement - out.println(e.getMessage()): null]]></system-out>
         <system-err><![CDATA[]]></system-err>
      </testcase>
      <system-out><![CDATA[2018-07-02 10:07:51 - [TEST_SUITE][PASSED] - HiUpTestSuite: Test Cases/JobFeedTest]]></system-out>
      <system-err><![CDATA[]]></system-err>
   </testsuite>
</testsuites>
